src/daemon/metric.h

```diff
@@ -64,16 +64,6 @@ typedef struct {
   char *value;
 } label_pair_t;
 
-/* label_t represents a key/value pair. It is similar to label_pair_t, except
- * that is has const fields. label_t is used in function arguments to prevent
- * the called function from modifying its argument. Internally labels are
- * stored as label_pair_t to allow modification, e.g. by targets in the "filter
- * chain". */
-typedef struct {
-  char const *name;
-  char const *value;
-} label_t;
-
```

Comment:
Why have there been two types `label_pair_t` and `label_t` when they have the same fields? But I guess it is ok to get rid of `label_t` and replace with `label_pair_t` in all occasions like you did.

Reply:
`label_t`'s fields were `const`. It was used as an argument to signal that the argument would not be modified. Almost all use-cases outside of unit tests had been removed though.